web/index.html

```diff
@@ -3,70 +3,8 @@
-
-        @font-face {
-          font-family: GTAmericaExp-Regular;
-          font-weight: 500;
-          font-style: normal;
-          src: url("https://www.expensify.com/font/GT-America-Standard-Medium.eot") format("embedded-opentype"), url("https://www.expensify.com/font/GT-America-Standard-Medium.woff") format("woff"), url("https://www.expensify.com/font/GT-America-Standard-Medium.woff2") format("woff2"); }
-
-        @font-face {
-          font-family: GTAmericaExp-Regular;
-          font-weight: 600;
-          font-style: normal;
-          src: url("https://www.expensify.com/font/GT-America-Standard-Medium.eot") format("embedded-opentype"), url("https://www.expensify.com/font/GT-America-Standard-Medium.woff") format("woff"), url("https://www.expensify.com/font/GT-America-Standard-Medium.woff2") format("woff2"); }
-
-        @font-face {
-          font-family: GTAmericaExp-Regular;
-          font-weight: 700;
-          font-style: normal;
-          src: url("https://www.expensify.com/font/GT-America-Standard-Bold.eot") format("embedded-opentype"), url("https://www.expensify.com/font/GT-America-Standard-Bold.woff") format("woff"), url("https://www.expensify.com/font/GT-America-Standard-Bold.woff2") format("woff2"); }
-
-        @font-face {
-            font-family: GTAmericaExp-RgIt;
-            font-weight: 400;
-            font-style: italic;
-            src: url("https://www.expensify.com/font/GT-America-Standard-Regular-Italic.eot") format("embedded-opentype"), url("https://www.expensify.com/font/GT-America-Standard-Regular-Italic.woff") format("woff"), url("https://www.expensify.com/font/GT-America-Standard-Regular-Italic.woff2") format("woff2"); }
-
-        @font-face {
-            font-family: GTAmericaExp-MdIt;
-            font-weight: 500;
-            font-style: italic;
-            src: url("https://www.expensify.com/font/GT-America-Standard-Medium-Italic.eot") format("embedded-opentype"), url("https://www.expensify.com/font/GT-America-Standard-Medium-Italic.woff") format("woff"), url("https://www.expensify.com/font/GT-America-Standard-Medium-Italic.woff2") format("woff2"); }
-
-        @font-face {
-            font-family: GTAmericaExp-BdIt;
-            font-weight: 700;
-            font-style: italic;
-            src: url("https://www.expensify.com/font/GT-America-Standard-Bold-Italic.eot") format("embedded-opentype"), url("https://www.expensify.com/font/GT-America-Standard-Bold-Italic.woff") format("woff"), url("https://www.expensify.com/font/GT-America-Standard-Bold-Italic.woff2") format("woff2"); }
-
-        * {
-            -webkit-font-smoothing: antialiased;
-            -moz-osx-font-smoothing: grayscale;
-        }
-
-        strong {
-            font-weight: 600;
-        }
-
         html,
```

Comment:
NAB do you know what's the deal with these other styles / why we wouldn't move them to a css file too?

Reply:
NAB so if there's a reason you left those styles in here then feel free to self-merge.

Reply:
Yep I only wanted to pull out things related to fonts here. When this is re-used we won't want to apply those `html` block styles or height 100% stuff.